.github/workflows/ci-components.yml

```diff
@@ -0,0 +1,89 @@
+name: CI - Components
+    
+on:
+  push:
+    branches:
+      - main
+  pull_request: {}
+
+env:
+  NODE_VERSION: '16.x'
+
+concurrency:
+   group: ci-components-${{ github.head_ref || github.ref }}
+   cancel-in-progress: true
+
+jobs:
+  test:
+    name: "Tests"
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          cache: yarn
+          cache-dependency-path: yarn.lock
+      - name: Install Dependencies
+        run: yarn install --frozen-lockfile
+      - name: Lint
+        run: yarn run lint
+        working-directory: packages/components
+      - name: Run Tests
+        run: yarn run test:ember:percy
+        working-directory: packages/components
+        env:
+          PERCY_TOKEN: ${{ secrets.PERCY_TOKEN_COMPONENTS }}
```

Comment:
@Dhaulagiri 2 remaining issues here:

(1) Percy not wired up correctly, gives this error https://github.com/hashicorp/design-system/runs/5545410323?check_suite_focus=true#step:6:20

```
[percy] Invalid snapshot options:
[percy] - scope: unknown property
```

Reply:
It looks like this was an existing problem prior to merging into the monorepo if you look at [this run from the old repo](https://github.com/hashicorp/flight/runs/5425624734?check_suite_focus=true#step:5:25).

Reply:
That's flight though, it was working here in design-system-components https://github.com/hashicorp/design-system-components/runs/5513674885?check_suite_focus=true#step:7:15

Can debug it here https://app.asana.com/0/1201630204387510/1201967845133306/f

Reply:
Ok, strangely Percy is working. What's incredibly odd is it did not work for ages but had stepped away for a bit https://percy.io/HashiCorp/ds-components/builds/16498978/unchanged/928471229?browser=firefox&viewLayout=overlay&viewMode=new&width=1280

Reply:
@amyrlam i'm not sure why percy wouldn't have been working, but the run you linked to initially was tests for flight here in the new repo which are showing the same message as a run from when it was in its own repo.